administrator/components/com_guidedtours/src/Extension/GuidedtoursComponent.php

```diff
@@ -28,6 +28,78 @@ class GuidedtoursComponent extends MVCComponent implements BootableExtensionInte
 {
     use HTMLRegistryAwareTrait;
 
+    /**
+     * The step type
+     *
+     * @since __DEPLOY_VERSION__
+     */
+    public const STEP_TYPE_NAMES = [
+        self::STEP_NEXT   => 'COM_GUIDEDTOURS_FIELD_VALUE_NEXT_STEP',
+        self::STEP_REDIRECT => 'COM_GUIDEDTOURS_FIELD_VALUE_REDIRECT',
```

Comment:
would it not be better for these three language keys to be
COM_GUIDEDTOURS_FIELD_VALUE_STEP_NEXT
COM_GUIDEDTOURS_FIELD_VALUE_STEP_REDIRECT
COM_GUIDEDTOURS_FIELD_VALUE_STEP_INTERACTIVE

Reply:
or 
COM_GUIDEDTOURS_FIELD_STEP_NEXT_VALUE

Reply:
Yeah... I will fix that